build/patches/Disable-third-party-origin-trials.patch

```diff
@@ -255,19 +256,5 @@ diff --git a/third_party/blink/renderer/core/origin_trials/origin_trial_context.
  }
  
  bool OriginTrialContext::IsFeatureEnabled(OriginTrialFeature feature) const {
-diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
---- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
-+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
-@@ -2272,6 +2272,10 @@
-       name: "TextFragmentTapOpensContextMenu",
-       status: {"Android": "stable"},
-     },
-+    {
-+      name: "ThirdPartyOriginTrials",
```

Comment:
If we remove this then `RuntimeEnabledFeatures::ThirdPartyOriginTrialsEnabled()` will not exist anymore; I think we should keep this, not just for the compilation error but for visibility.

Reply:
> not just for the compilation error

but that's what I want, I want a build error.
the alternative is to periodically check that patch to see if the flag is otherwise used, instead so it is the compiler that warns us.